README.md

```diff
@@ -5,6 +5,8 @@ The script is a new edition of an earlier [mkstage4 script](https://github.com/g
  
 More information on mkstage4 can be found on its own Chymeric Tutorials article: [mkstage4 - Stage 4 Tarballs Made Easy](http://tutorials.chymera.eu/blog/2014/05/18/mkstage4-stage4-tarballs-made-easy/). 
 
+Chinese Introduction [中文说明](http://liuk.io/blog/gentoo-stage4)
```

Comment:
If you want to link from here to the external docs, please also link back to here from the external docs.

Reply:
Yes. The page linked back.

Reply:
The example is deleted.